resources/jira/days_in_sprint.py

```diff
@@ -0,0 +1,180 @@
+from collections import defaultdict
+from typing import Dict, List, Tuple, Union
+import os
+from datetime import date
+
+from pandas import DataFrame, read_csv
+
+import matplotlib.pyplot as plt
+
+from jira import JIRA
+from jira.resources import Issue, Sprint, Board
+from jira.client import ResultList
+
+
+class ProgressMonitor:
+    PAGE_SIZE = 50
+
+    lut = {"new": 0, "todo": 1, "in progress": 2, "review": 3, "ready for qa": 4, "done": 5}
+
+    def __init__(self) -> None:
+        api_key = os.environ.get("API_KEY", "")
```

Comment:
Same remark as for JiraReport, a suggestion is to provide these parameters to the constructor, or provide the JIRA object.